segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -260,7 +260,11 @@ private void cleanupDirectories() throws IOException {
     private void cleanupDirectories(HashMap<?, File> toDelete) throws IOException {
         for (File dir : toDelete.values()) {
             log.info("Cleaning up " + dir);
-            FileUtils.deleteDirectory(dir);
+            try {
+                FileUtils.deleteDirectory(dir);
+            } catch (IOException e) {
+                log.info("Failed deleting directory: {} - {}", dir, e);
```

Comment:
Sorry, I missed to tell that you don't need '{}' for the exception. The logger will print the message and the stacktrace anyway.